contract-testing/pact-messaging/provider/src/main/kotlin/provider/messaging/PublishEventViaMessagingFunction.kt

```diff
@@ -0,0 +1,16 @@
+package provider.messaging
+
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import provider.books.BookCreatedEvent
+import provider.events.PublishEventFunction
+
+@Service
+class PublishEventViaMessagingFunction : PublishEventFunction {
```

Comment:
not needed, since it is replaced in tests

Reply:
done